roles/ddns_updater/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Create DDNS Updater Directories
+  file:
+    path: "{{ ddns_updater_data_directory }}"
+    state: directory
+
+- name: Check config is defined
+  fail: msg="No 'ddns_updater_config' defined."
+  when: ddns_updater_config is not defined
+
+- name: Generate config.json
+  copy:
+    content: "{{ ddns_updater_config | to_nice_json }}"
+    dest: "{{ ddns_updater_data_directory }}/config.json"
+
+- name: Generate dns docker environment
+  set_fact:
+    ddns_update_env: |
+      TZ: "{{ ansible_nas_timezone }}"
```

Comment:
`TZ` should be in the task file as its required. it shouldn't be configurable

Reply:
I put everything back directly to env of the container